codegen/smithy-ruby-codegen-test/integration-specs/types_spec.rb

```diff
@@ -58,5 +58,21 @@ module Types
         expect(unknown_union.to_h).to eq({ unknown: { key: 'key', value: 'value' } })
       end
     end
+
+    describe DefaultsTestInput do
+      subject { DefaultsTestInput.new }
```

Comment:
I believe subject is implicitly defined already

Reply:
Ah, yes it is.